transmogrifier/sources/json/aardvark.py

```diff
@@ -349,17 +349,41 @@ def get_publication_information(source_record: dict) -> list[str]:
         return publication_information
 
     @staticmethod
-    def get_rights(source_record: dict) -> list[timdex.Rights]:
+    def get_rights(source: str, source_record: dict) -> list[timdex.Rights]:
         """Get values from source record for TIMDEX rights field."""
         rights = []
 
         if "dct_accessRights_s" in source_record:
+            kind_access_rights = "Access"
+            kind_access_to_files = "Access to files"
```

Comment:
I get `"Access to files"` but I'm unclear about the difference here, what is `"Access"` referring to? And it looks like `kind_access_rights` is only used once so does it need to be a variable?

Reply:
I am not sure of the intention of `kind = "Access"`, but all it seems to do is retrieve the value from the Aardvark field [dct_accessRights_s](https://opengeometadata.org/ogm-aardvark/#access-rights). For the MITAardvark transformer, the `get_rights()` field method essentially pulls from fields under the [Aardvark Rights category](https://opengeometadata.org/ogm-aardvark/#rights).

@ehanson8 and @ghukill - What do you think of updating `kind="Access"` -> `kind="Access rights"`?

Reply:
Maybe a better way of stating my question is: is there actually a difference between `Access` and `Access to files`? There may not be but if we have 2 different strings, the user may think there's a difference

Reply:
I agree with @ehanson8 about "Access" vs "Access to files" being very similar and potentially confusing.  But the language for the values, and the language for that kind both come directly from the UI mockup:

<img width="250" alt="7328473a-bcf5-4d91-9fbb-6458922eef72" src="https://github.com/MITLibraries/transmogrifier/assets/1753087/b5d48200-6883-456d-86d7-458f985f50c4">

While maybe we could make the kind something like "Download Access", we'd need to confirm with EngX that the actual "kind" value could differ from the text on the UI.  I imagine it could, but unsure.

OR, we could just continue with "Access to files" (which does _kind_ of suggest downloading), and then iterate later if determined confusing.

As far as the string values go (e.g. "Not owned by MIT"), I'd imagine those are good as-is because they _will_ render directly on the page via the aggregation.

Reply:
I'm OK with that as is given the draft status and I get it now seeing the UI, but to me `Access to files` seems like the only type of access users would care about. But this is better sorted out in inter-team discussions later!

Reply:
Some context (this may no longer be relevant, I haven't checked to see what you all decided on in the code).

In general, field values _can_ be manipulated between OpenSearch and TIMDEX API (or TIMDEX API and TIMDEX UI) when we have a good reason to do so, but when we find ourselves wanting to do that we should probably pause and ask why. If UX is suggesting a label in the UI is the best way to describe something from a user perspective, is there a reason an API user would not find the same language helpful? Or is there a reason we want to introduce potential future confusion between what is in OpenSearch and TIMDEX API?

So before forking our language across tiers we should reach out to UX and explain the situation and ask if they would consider language we feel is more specific or accurate. If there is a good reason to make a distinction between how we refer to data in TIMDEX Spec and TIMDEX API/TIMDEX UI, we can definitely support it... we just need to be sure that's what we want :)

Reply:
I better see your point @ehanson8, that _conceptually_ what's even the difference...  And, agreed about inter-team discussing, which [this slack thread might tease out](https://mit-libraries.slack.com/archives/C05UK9PDNBW/p1706195251372659).

AND, one of the open questions for GDT-138 is whether this will even be a `Rights` entry, or maybe a dedicated field, so might all be moot if it's a new field.